controller/search.js

```diff
@@ -21,6 +21,11 @@ function setup( backend, query ){
       cmd.type = req.clean.layers;
     }
 
+    // set type if input suggests targeting a layer(s)
```

Comment:
It would be nice to set ```clean.default_layers``` at the ```_layers``` sanitiser - that way we wont have to set ```default_layers_set``` flag. Furthermore, we can just fall back to the ```default_layers``` when ```target_layers``` and ```clean.layers``` are not set. And that can be done in a one liner in the search controller if required :D 

But this shouldn't delay this PR from getting merged. I think that layer rewrite should be a tech-debt-y issue in the API.. https://github.com/pelias/api/issues/170